[2024-06-24 12:55:30,691] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-10-07T00:00:00+00:00 [queued]>
[2024-06-24 12:55:30,740] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-10-07T00:00:00+00:00 [queued]>
[2024-06-24 12:55:30,752] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 12:55:30,755] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 12:55:30,756] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 12:55:30,899] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-10-07T00:00:00+00:00
[2024-06-24 12:55:30,934] {standard_task_runner.py:52} INFO - Started process 30681 to run task
[2024-06-24 12:55:30,985] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-10-07T00:00:00+00:00', '--job-id', '1296', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpvuh8dzam', '--error-file', '/tmp/tmpw6fxa25b']
[2024-06-24 12:55:31,060] {standard_task_runner.py:77} INFO - Job 1296: Subtask second_task
[2024-06-24 12:55:31,395] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-10-07T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 12:55:31,670] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-07T00:00:00+00:00
[2024-06-24 12:55:31,675] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 12:55:31,689] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 12:55:31,750] {bash.py:169} INFO - Output:
[2024-06-24 12:55:31,767] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 12:55:31,778] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 12:55:32,231] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211007T000000, start_date=20240624T125530, end_date=20240624T125532
[2024-06-24 12:55:32,642] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 12:55:32,719] {local_task_job.py:146} INFO - Task exited with return code 0
